R-package/cran-comments.md

```diff
@@ -20,7 +20,7 @@ Submitted with the following comment:
 
 > Thank you very much for your time and consideration.
 
-## v3.3.3 - Submission 1 - (October 10, 2022)
+## v3.3.4 - Submission 1 - (October 10, 2022)
```

Comment:
I think this one is correct with 3.3.3 ([ref](https://github.com/microsoft/LightGBM/pull/5525#issuecomment-1273516728))
```suggestion
## v3.3.3 - Submission 1 - (October 10, 2022)
```

Reply:
blegh yeah you're right, maybe that was from a bad merge conflict resolution. Thanks for catching it!